src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,63 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Page Block Magnification
+
+Page blocks have a magnification option that allows the user to zoom in order to inspect the details of the block in distraction-free mode.
+By default, block magnification is disabled.
+When set, blocks that have this configuration will have a zoom-in icon in the top right corner of the block, allowing the user to magnify the block upon clicking on the icon.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/magnification-icon.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 0,
+    "y": 0,
+    "w": 1920,
+    "h": 1080
+  },
+  "annotations": [{
+    "kind": "box",
+    "padding": "lg",
+    "x": 1370,
+    "y": 82,
+    "w": 24,
+    "h": 24
+  }]
+}
+----
+
+=== Configuring Page Block Magnification
+
+To display the magnification icon and enable block magnification, navigate to the page builder, choose a block, and go to its general tab.
+There you will find the block magnification options as a select dropdown.
+
+image:compose-configuration/enable-magnification.png[Page Block Magnification Options, role="data-zoomable"]
```

Comment:
Let's give this image some annotations.
Blank images when you're providing instructions generally don't sit well with me